Task2/Task2/Task2/Products/Bolt.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Task2.Exceptions;
+
+namespace Task2.Products
+{
+    public class Bolt : Product
+    {
+        public Bolt(string productName, double purchaseCost, double margin, int numberOfUnits)
+            : base(productName, purchaseCost, margin, numberOfUnits)
```

Comment:
If you have more than 2 parameters in a method, you should move them to new lines.

Reply:
fixed